src/controllers/reset-password.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable consistent-return */
+const { User } = require('models');
+const Joi = require('joi');
+const { scryptSync } = require('crypto');
+const jwt = require('jsonwebtoken');
+const { cryptoLink, decryptLink } = require('./crypto');
+// eslint-disable-next-line no-useless-escape
+const passwordRegex = /^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#\$%\^&\*])(?=.{8,})/;
+const setResetPassword = async (req, res) => {
+  try {
+    const schema = Joi.object({ email: Joi.string().email().required() });
+    const { error } = schema.validate(req.body);
+    if (error) return res.status(201).send(error.details[0].message);
+
+    const user = await User.findOne({ where: { email: req.body.email } });
+    if (!user)
+      return res
+        .status(201)
+        .send(
+          'Sorry we can’t find an account with the entered email address. Please try again or create new account.'
+        );
+
+    const token = {
+      userId: user.id,
+      token: jwt.sign({ email: req.body.email }, 'Stack', {
+        expiresIn: '1800s',
+      }),
+    };
+    const encryptedToken = cryptoLink(token.token);
+    // eslint-disable-next-line max-len
+    const link = `http://reverifi-api.docker/api/password-reset/${user.dataValues.id}/${encryptedToken}`;
+    res.send(link);
```

Comment:
i think this link should be sent to the user by email, right ?
also pleae use the `response` utility to return the response data
```suggestion
    res.json(response({ data: { link } }));
```